src/http-client/util.ts

```diff
@@ -54,7 +60,8 @@ function getTxnIdFromURL(urlString: string): string {
   const url = new URL(urlString);
   const value = url.searchParams.get('authIndexValue') || '';
   const parser = new DOMParser();
-  const doc = parser.parseFromString(value, 'text/xml');
+  const decodedValue = decodeURIComponent(value);
```

Comment:
Found a bug in the (currently unsupported) decoding of IG's XML response by accident :)